packages/p2p/src/components/app-content.jsx

```diff
@@ -37,6 +39,20 @@ const AppContent = () => {
         return <AdvertiserPage />;
     }
 
+    if (general_store.props.setP2POrderProps) {
+        general_store.props.setP2POrderProps({
```

Comment:
I tried putting this in a `useEffect` but it would still not detect `setP2POrderProps` function

Reply:
`setP2POrderProps` is from mobx. It is wrapped in a `Proxy` so React's useEffect wouldnt know what changed. You can use a reaction for this BTW

Reply:
ooo thats why its saying `setP2POrderProps` is undefined when I put it inside the useEffect 😓